### DISCUSSION TITLE
Are you planning to make masonry style grid layout using Grid?

### DISCUSSION THREAD

**Discussion by KoushikMVN** [ant-design/ant-design-mobile#4379] [Ideas]
Are you planning to make masonry style grid layout using Grid?

**Comment by awmleer** [comment]
Masonry style? I'm a little bit confused... I searched MDN and found this one: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Masonry_Layout
Is this what you expected?

**Reply by KoushikMVN** [reply]
Yes
